packages/ui/test/overview/components/WorkingGroupsOverview.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { createType } from '@joystream/types'
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+import { MemoryRouter } from 'react-router'
+
+import { ApiContext } from '@/common/providers/api/context'
+import {
+  seedWorkingGroups,
+  seedMembers,
+  seedOpening,
+  seedOpeningStatuses,
+  seedApplication,
+  seedWorker,
+} from '@/mocks/data'
+import { WorkingGroupsOverview } from '@/overview/components/WorkingGroupsOverview/WorkingGroupsOverview'
+
+import { MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+import { APPLICATION_DATA, OPENING_DATA, WORKER_DATA } from '../../_mocks/server/seeds'
+import { stubApi, stubConst } from '../../_mocks/transactions'
+
+describe('UI: Working groups overview', () => {
+  const server = setupMockServer({ noCleanupAfterEach: true })
+  const api = stubApi()
+
+  describe('General info', () => {
+    beforeAll(() => {
+      seedMembers(server.server)
+      seedWorkingGroups(server.server)
+      seedOpeningStatuses(server.server)
+      seedOpening(OPENING_DATA, server.server)
+      seedApplication(APPLICATION_DATA, server.server)
+      seedWorker(WORKER_DATA, server.server)
+      seedWorker(
+        {
+          ...WORKER_DATA,
+          id: 'forumWorkingGroup-2',
+        },
+        server.server
+      )
+    })
+
+    beforeEach(async () => {
+      stubConst(api, 'forumWorkingGroup.maxWorkerNumberLimit', createType('u32', 10))
+      renderComponent()
+    })
+
+    it('Displays number of working groups', async () => {
+      expect((await screen.findByText('workingGroups.workingGroups')).previousSibling?.textContent).toBe('4')
+    })
+
+    it('Displays number of workers', async () => {
+      expect((await screen.findByText('workingGroups.workers')).previousSibling?.textContent).toBe('2')
+    })
+
+    it('Displays total budget', async () => {
+      expect((await screen.findByText('workingGroups.totalBudget')).previousSibling?.firstChild?.textContent).toBe(
+        '14,097'
```

Comment:
IMO this approach will cause problems in long term. The issue here is that you expect an outcome based on the `seedWorkingGroups` which populate component with json mock, taking it further I think any working group mock generation will result in different budget for each group, so the test will require a change after it. 

`budget: randomFromRange(1000, 5000),` <- this is how budget is determined in mock generation

I would take approach of mocking the hook and steer data manually.